### ISSUE TITLE
About Section

### ISSUE THREAD

**Issue by holly-johnson** [KayleeZnamenacek/kaylee-portfolio-site#17]
I like your image!

Is this content supposed to be 3 paragraphs? You should use different p tags instead of BRs. 

<img width="698" alt="Screen Shot 2020-05-10 at 8 36 39 PM" src="https://user-images.githubusercontent.com/15253544/81517187-ade4f300-92ff-11ea-9851-ca5fc7ab981e.png">

**Comment by KayleeZnamenacek** [comment]
Yeah, I was playing around with how it was organized and quickly put in br tags. thanks for reminding me. I'll change them to p tags!
